src/platform/packages/shared/kbn-esql-composer/src/utils/formatters.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export function escapeColumn(column: string) {
+  return `\`${column}\``;
+}
+
+export const formatValue = (value: any) => {
+  return typeof value === 'string' ? `"${value}"` : value;
+};
+
+export const formatColumn = (column: string) => {
```

Comment:
We are using this only in SORT. Why??
Also why do you check only when it starts with ??. I am assuming that you are testing for named params but they can also start with one ?

Reply:
This was to escape field names, but I believe this will be something the composer can defer to ast